tools/tslint-rules/noViewEncapsulationRule.ts

```diff
@@ -35,23 +35,23 @@ class Walker extends Lint.RuleWalker {
     this._enabled = fileGlobs.some(p => minimatch(relativeFilePath, p));
   }
 
-  visitClassDeclaration(node) {
+  visitClassDeclaration(node: any) {
```

Comment:
Don't you want to use types here?  e.g `ts.ClassDeclaration`? (also below)

Reply:
I can't definitely use it for the `node`, but the `expression` below doesn't seem to be in the typings.

Reply:
Not really something that is too important. For the `expression` you could use `ts.CallExpression`, but it will not have types for the `arguments[0].properties`.

Reply:
Yeah eventually the `arguments` have to be cast to `any`.